src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/product/update.yaml

```diff
@@ -14,9 +14,12 @@ twig_hooks:
                     method: 'PUT'
 
         'sylius_admin.product.update.content.header.title_block.actions':
+            show:
+                template: '@SyliusAdmin/shared/crud/update/content/header/title_block/actions/show.html.twig'
+                priority: 300
             view_in_store:
                 template: '@SyliusAdmin/product/common/header/title_block/actions/view_in_store.html.twig'
-                priority: 0
+                priority: 400
             manage_variants:
                 template: '@SyliusAdmin/product/common/header/title_block/actions/manage_variants.html.twig'
                 priority: 300
```

Comment:
`Priority: 300, 400, 300` Could we keep this in order?

Reply:
I believe I merged it incorrectly after the rebase. I will correct it 🔧